packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelUtils.ts

```diff
@@ -18,14 +19,26 @@ export function getWarningTokenDescription(warningTokenType: number) {
   }
 }
 
+function sanitizeTokenSearchParams(
+  tokenAddress: string | null
+): string | undefined {
+  if (!tokenAddress) {
+    return undefined
+  }
+  if (utils.isAddress(tokenAddress)) {
+    return tokenAddress
+  }
+  return undefined
+}
+
 export function useTokenFromSearchParams(): {
```

Comment:
I think we should kill this hook, it adds confusion. Everything should be handled in `useSelectedToken`